### PR TITLE
Minor changes to verifications

### PR DESCRIPTION
**Pull request by nambrot**
Fixes up minor things about verification

### CODE REVIEW

File: typescript/infra/scripts/verify.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ async function main() {
   // from solidity/core/hardhat.config.ts
   const compilerOptions: CompilerOptions = {
     codeformat: 'solidity-single-file',
-    compilerversion: 'v0.8.13+commit.07a7930e',
+    compilerversion: 'v0.8.16+commit.07a7930e',
```
Comment (yorhodes) [inline typescript/infra/scripts/verify.ts:35]: we should just add an override flag for this too

File: typescript/sdk/src/deploy/verify/ContractVerifier.ts [hunk 2]
```diff
@@ -102,6 +102,7 @@ export class ContractVerifier<Chain extends ChainName> extends MultiGeneric<
           this.logger(`Proxy verification failed, try manually?`);
           return;
         default:
+          this.logger(`Verification failed for some unknown reason`, result);
```
Comment (yorhodes) [inline typescript/sdk/src/deploy/verify/ContractVerifier.ts:105]: doesn't the error throw this?

**Review by yorhodes — APPROVED** [review]

File: typescript/sdk/src/deploy/verify/ContractVerifier.ts (hunk 2 above)
Comment (nambrot) [reply typescript/sdk/src/deploy/verify/ContractVerifier.ts:105]: It includes the result

File: typescript/infra/scripts/verify.ts (hunk 1 above)
Comment (nambrot) [reply typescript/infra/scripts/verify.ts:35]: yeah probably, but leave that for a future PR

**Pull request merged**